### ISSUE TITLE
Normalizing possible wrong axis

### ISSUE THREAD

**Issue by lix4** [alireza-nasiri/SoundCLR#4]
I got a question. You specify dim=0 for every F.normalize(). Is it normalizing along with the batch size axis? The shape of `y_rep`  is `B x 2048`. If you want to normalize each embedding on a unit hypersphere, should it be through dim=1?
